cmd/tink-server/main.go

```diff
@@ -188,12 +188,9 @@ func NewRootCommand(config *DaemonConfig, logger log.Logger) *cobra.Command {
 			}, errCh)
 
 			httpServer.SetupHTTP(ctx, logger, &httpServer.Config{
-				CertPEM:               cert,
-				ModTime:               modT,
-				GRPCAuthority:         config.GRPCAuthority,
-				HTTPAuthority:         config.HTTPAuthority,
-				HTTPBasicAuthUsername: config.HTTPBasicAuthUsername,
-				HTTPBasicAuthPassword: config.HTTPBasicAuthPassword,
```

Comment:
Is the basic auth only limited to json rpc gateway?

Reply:
Yep, https://github.com/tinkerbell/tink/pull/583/files#diff-441184fd3cd157ac00ab9c45d677b03cf32b3113097d24800ea9e5ee60fa965dL82 is where the auth is used and its for stuff registered off of the grpc http mux https://github.com/tinkerbell/tink/pull/583/files#diff-441184fd3cd157ac00ab9c45d677b03cf32b3113097d24800ea9e5ee60fa965dL50 only. The other http paths are auth free.

Reply:
which duh, means I have more code I can delete :D